internal/pkg/mapper/orchestrator/remote_load_test.go

```diff
@@ -543,6 +563,7 @@ WARN  Warning: invalid orchestrator config "branch:123/component:keboola.orchest
   - found cycles in phases "dependsOn"
     - 3 -> 4 -> 3
     - 1 -> 2 -> 1
+    - 5 -> 8 -> 7 -> 6 -> 5
```

Comment:
Pridal som komplexnejsi test na vypisovanie cyklov, ... zistil som, ze sa vypisuju opacne -> fixnute ..
V riadkoch vyssie to bolo jedno, lebo odpredu, aj odzadu je vysledok rovnaky.